frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/AdmittedCase.js

```diff
@@ -310,15 +310,17 @@ const AdmittedCases = () => {
     [applicationData, onBehalfOfuuid]
   );
 
-  const isDelayApplicationPending = useMemo(
-    () => {
-      return Boolean(
+  const [isDelayApplicationPending, setIsDelayApplicationPending] = useState(false);
+
+  useMemo(() => {
```

Comment:
this is an odd combination of state and memo.

are you sure you need useMemo here instead of useEffect ?

if the behaviour is wrong with useEffect, we can move ahead with this, but useMemo should not have sideEffects like that @nitish-beehyv